### DISCUSSION TITLE
Add to cart pop up

### DISCUSSION THREAD

**Discussion by lennertfabriek** [bootscore/bootscore#379] [Q&A]
Hi! I want this popup model after add to cart (see image)

<img width="673" alt="Schermafbeelding 2023-01-31 om 12 54 42" src="https://user-images.githubusercontent.com/78724181/215753476-7c6befd3-d664-48fc-a43c-f37647c10a8f.png">

I already:
- disabled the ajax mini cart
- Made a pop up after add to cart

But of course I need to make this work with error logs too. And the remaining value before free shipping would be nice (but no prior). 

Did someone made this already?

**Comment by lennertfabriek** [comment]
So when error, don't display the pop up but normal error notice. When succeed, open pop up (something like the ajax-add-to-cart.php code, but really don't know how to use this in my case)

**Comment by justinkruit** [comment]
You could do this with some other javascript, if you check if the body triggers "added_to_cart" you could then show the popup. You'd need to calculate the price tho, you could do this by sending an ajax call when triggering the added to cart to answer back with the modal content.

**Comment by lennertfabriek** [comment]
Thanks! I use the following:
$( document.body ).on( 'added_to_cart', function(e) {
       $(".productpopup-wrapper").addClass("show");
        e.stopPropagation()
        });
}

But it triggers every time I press the add to cart button. So also when it doesn't add
